bip-musig2.mediawiki

```diff
@@ -75,14 +75,14 @@ Similarly, the test vectors that exercise the unimplemented features should be r
 === General Signing Flow ===
 
 The signers start by exchanging public keys and computing an aggregate public key using the ''KeyAgg'' algorithm.
-Whenever they want to sign a message, the basic order of operations to create a multi-signature with the specification is as follows:
+Whenever they want to sign a message, the basic order of operations to create a multi-signature is as follows:
 
 '''First broadcast round:'''
 The signers start the signing session by running ''NonceGen'' to compute ''secnonce'' and ''pubnonce''<ref>We treat the ''secnonce'' and ''pubnonce'' as grammatically singular even though they include serializations of two scalars and two elliptic curve points, respectively. This treatment may be confusing for readers familiar with the MuSig2 paper. However, serialization is a technical detail that is irrelevant for users of MuSig2 interfaces.</ref>.
 Then, the signers broadcast their ''pubnonce'' to each other and run ''NonceAgg'' to compute an aggregate nonce.
 
 '''Second broadcast round:'''
-At this point, every signer has the required data to sign, which, in the specification, is stored in a data structure called [[#session-context|Session Context]].
+At this point, every signer has the required data to sign, which, in the description here, is stored in a data structure called [[#session-context|Session Context]].
```

Comment:
The word "here" is maybe confusing. It doesn't refer to the paragraph or section. Do you mean "description below"?